packages/@sanity/components/src/toggles/styles/Switch.css

```diff
@@ -38,20 +37,17 @@
   display: flex;
 }
 
-.indeterminate {
-  composes: root;
-}
-
 .switchWrapper {
   position: relative;
+  vertical-align: center;
```

Comment:
This should be `vertical-align: middle` (however, I tend to use `vertical-align: top` for non-text nodes and do vertical alignment using other techniques).